infra/util/transform_file_to_string_template.cpp.conf

```diff
@@ -0,0 +1,12 @@
+#include "${name}.hpp"
+#include <array>
+
+namespace ${namespace}
+{
+    namespace
```

Comment:
Is there a specific reason to have an anonymous namespace inside another namespace?

Reply:
I think so. The ${name}Storage variable is not public to other files, so I'd like to have it in an unnamed namespace. And I like my unnamed namespaces nested in the namespaces in which they are used.

Reply:
Fair enough. I've just always put my anonymous namespaces at the top of the file outside any named namespace.